src/Microsoft.AspNetCore.ResponseCaching/Internal/IResponseCacheKeyProvider.cs

```diff
@@ -3,9 +3,9 @@
 
 using System.Collections.Generic;
 
-namespace Microsoft.AspNetCore.ResponseCaching
+namespace Microsoft.AspNetCore.ResponseCaching.Internal
 {
-    public interface IResponseCacheKeyProvider
+    internal interface IResponseCacheKeyProvider
```

Comment:
Should we clear the usage of namespace `*.Internal`? Because I remember not long ago, we decide to use this namespace to hold those types we design to be internal use but still want to offer users the convenience of customization. We left them `public` but keep them in `Internal` namespace so as to indicate that the type may be broken in minor upgrade.

Are we still doing this?